src/components/Header.tsx

```diff
@@ -173,24 +173,21 @@ export const Header: React.FC = () => {
                   as={NavLink}
                   className={classnames(styles["nav-link"], {
                     [`fw-bold ${styles.active}`]: useIsActive(
-                      ["/gallery", "/oer"],
```

Comment:
When removing routes, it's best to add redirects to pages that do exist. You can do this via the `redirects` option in `next.config.mjs` at the root of this repository. Let me know if you'd like help with that.

Reply:
done